src/language/errors.md

```diff
@@ -0,0 +1,14 @@
+# Error handling
+
+Error handling in Rust is done via the `Result` type (see the structs and
+enums section). But on top of explicit pattern-matching, hacspec also
+supports the popular `?` operator to quickly perform an early return and
```

Comment:
We should also mention that early returns and the return statement itself are not allowed otherwise.